pkg/kubelet/kubelet.go

```diff
@@ -2366,6 +2371,9 @@ func (kl *Kubelet) setNodeStatus(node *api.Node) error {
 		if !networkConfigured {
 			messages = append(reasons, "network not configured correctly")
 		}
+		if !selinuxConfigured {
+			messages = append(reasons, "SELinux is required by the cluster but not enabled on this node")
```

Comment:
I would make this a little more verbose to match the phrasing of the name of the capability:

"SELinux integration is enabled in the cluster but SELinux is not enabled on this node.  See /etc/sysconfig/selinux."